README.md

```diff
@@ -29,7 +31,7 @@ Download a release package for your architecture from the [project's GitHub rele
 #### Installing Finch via [brew](https://brew.sh/)
 
 ```sh
-$ brew install --cask finch
+brew install --cask finch
```

Comment:
I think someone specifically added these `$` in a prior PR to indicate they are commands to run from a terminal. I've seen other projects use the `$` and not use the `$` to indicate this, just wondering what you think? I'm ok with removing them, but maybe there's a good reason to keep them?